version-map.txt

```diff
@@ -3,3 +3,4 @@ v7.0.0
 v7.0.1
 v7.0.2
 v7.0.3
```

Comment:
```suggestion
v7.0.4
```
Only the latest version is needed here. I don't have this documented anywhere: really the only reason to include multiple versions here is (a) if you change the underlying python image and want have the change available for multiple Sopel images, and (b) like your previous PR, we were missing some versions and want to get them built in one go. In this case, neither apply, so just the latest version should be present.